src/main/java/mate/academy/model/ShoppingCart.java

```diff
@@ -0,0 +1,61 @@
+package mate.academy.model;
+
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.MapsId;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "shopping_carts")
+public class ShoppingCart {
+    @Id
+    private Long id;
+    @OneToMany(cascade = CascadeType.MERGE)
+    @JoinTable(name = "shoppingCarts_tickets",
+            joinColumns = @JoinColumn(name = "shoppingCarts_id"),
+            inverseJoinColumns = @JoinColumn(name = "tickets_id"))
```

Comment:
I propose to you naming like that:
```suggestion
    @JoinTable(name = "shopping_carts_tickets",
            joinColumns = @JoinColumn(name = "shopping_carts_id"),
            inverseJoinColumns = @JoinColumn(name = "tickets_id"))
```